tests/end2end/features/misc/conftest.py

```diff
@@ -23,3 +23,33 @@ def check_clipboard(clipboard, text):
 @bdd.then(bdd.parsers.parse("The primary selection should contain {text}"))
 def check_primary(clipboard, text):
     assert text in clipboard.text(mode=QClipboard.Selection)
+
+
+@bdd.then(bdd.parsers.parse("The clipboard should contain image {image}"))
+def check_clipboard_image(clipboard, image):
+    assert clipboard.pixmap(mode=QClipboard.Clipboard).toImage() == QImage(image)
```

Comment:
I asssume this test fails for some reason under qt 5.9 and 5.10. I would be happy to loosen this to `should contain any image` where we just ensure that we successfully saved an image to the clipboard without comparing the content directly.

Reply:
This seems to work. Changed that in 0fb4369